realm-jni/build.gradle

```diff
@@ -164,8 +164,13 @@ targets.each { target ->
 
     task "copyAndroidJni${target.name.capitalize()}"(type: Copy) {
         dependsOn "buildAndroidJni${target.name.capitalize()}"
-        from "${projectDir}/src/librealm-jni-${target.name}.so"
+        from "${projectDir}/src/librealm-jni-${target.name}-stripped.so"
         into "${projectDir}/../realm/src/main/jniLibs/${target.jniFolder}"
+        rename "librealm-jni-${target.name}-stripped.so", 'librealm-jni.so'
+
+        // Store the unstripped version
+        from "${projectDir}/src/librealm-jni-${target.name}.so"
+        into "${projectDir}/../realm/src/main/jniLibs-unstripped/${target.jniFolder}"
```

Comment:
Wouldn't it be better to store this in the build directory somewhere? Probably under `/build/outputs/jniLibs-unstripped`?